src/reducers/CurriculaSkillsReducer.ts

```diff
@@ -0,0 +1,43 @@
+import { Skill } from '../models/skill'
+import { skillPillTypes } from '../action-mappers/skill-pill-action-mapper'
+import { ISkillState } from './index'
+import { Curriculum } from '../models/curriculum'
+import { Category } from '../models/category'
+
+const initialState : ISkillState = {
+    skills: [new Skill(0,'',new Category(0,''))],
+    newCurriculum: new Curriculum(0,'',[])
+}
```

Comment:
Why is the making of a new curriculum on the skill state? Does it even need to be in redux? Can we just add the new on to the list of all curricula once we successfully make a new one?